api/types/src/convert.rs

```diff
@@ -145,7 +145,7 @@ impl<'a, R: MoveResolver + ?Sized> MoveConverter<'a, R> {
                 (info, payload, events).into()
             },
             BlockMetadata(txn) => (&txn, info, events).into(),
-            StateCheckpoint(_) => {
+            StateCheckpoint(_) | BlockEpilogue { .. } => {
```

Comment:
@grao1991 this should make it mostly a no-op for the ecosystem

Reply:
hmm.... I have concern about this. Do you plan to change this in the future? Isn't it more confusing if for the same version, the transaction is StateCheckpoint now but BlockEpilogue in a later version?

Reply:
We can change it via a feature flag in 1.10?

theoretically, I can get it now with feature flag, and try to release it to devnet with a new transaction type, and then see if anything breaks, to see how to release to mainnet?

Reply:
I'd like to see other people's opinion here. It would be weird and inefficient for API to depend on an onchain feature flag.